buildtools/pip/bindata/bindata.go

```diff
@@ -1,8 +1,6 @@
-// Code generated by go-bindata.
+// Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
These edits are probably a remnant from a previous `make` on a different branch. Rebasing onto master should solve your problem, although removing it from the PR accomplishes the same thing.